src/Uno.UI/UI/Xaml/Controls/AppBar/AppBarButton.Partial.cs

```diff
@@ -794,7 +795,7 @@ private void UpdateToolTip()
 
 					var toolTipFormatString = DXamlCore.Current.GetLocalizedResourceString("KEYBOARD_ACCELERATOR_TEXT_TOOLTIP");
 
-					this.SetValue(ToolTipService.ToolTipProperty, string.Format(toolTipFormatString, labelText, keyboardAcceleratorText));
+					this.SetValue(ToolTipService.ToolTipProperty, string.Format(CultureInfo.InvariantCulture, toolTipFormatString, labelText, keyboardAcceleratorText));
```

Comment:
This is presented to the user, it needs to be `CurrentUICulture`.

Reply:
```suggestion
					this.SetValue(ToolTipService.ToolTipProperty, string.Format(CultureInfo.CurrentUICulture, toolTipFormatString, labelText, keyboardAcceleratorText));
```